app/Resources/views/includes/base.html.twig

```diff
@@ -23,7 +26,30 @@
         <link rel="stylesheet" href="{{ asset('css/base.css') }}">
     </head>
     <body>
-        {% block body %}{% endblock %}
+        {# Navbar #}
+        {% block navbar %}{% include 'includes/navbar.html.twig' %}{% endblock %}
+        <div class="container-fluid {% block containerCssClass %}{% endblock %}">
+            {% block body %}
```

Comment:
Das können wir nicht auslagern?

Reply:
Ich verstehe, dass du die base.html.twig klein halten möchtest, jedoch wird es langsam etwas verwirrend mit den includes und es hat neulich schon nicht mehr auf Anhieb funktioniert. Ggf. muss hierfür die gesamte Struktur etwas umgebaut werden, damit wir nur noch aufwärts gehende extends und abwärts gehende includes haben.
Ich sehe die Navbar als Elementares Stück der Seitenstruktur und würde sie erstmal in der base.html.twig belassen. Du darfst aber ein neues Issue aufmachen und mir zuweisen, wenn ich das noch einmal aufdröseln soll.

Reply:
Ahh ich erinnere mich :) Jop dann lassen wir die erst mal drin.